docs/en/guides/compile/How-to-compile.md

```diff
@@ -2,7 +2,7 @@
 
 ## Go version
 
-Go version `1.17` or higher is supported for compilation.
+Go version `1.18` or higher is supported for compilation.
```

Comment:
Can you confirm this is correct? Usually I found higher Go version cannot compile lower Go version project seamlessly, usually some modifications are needed

Reply:
I have checked the GO 1.19.2, It works well.
![image](https://user-images.githubusercontent.com/3417650/194690227-2b160e82-3a6f-4554-b49f-6cde28c2325b.png)

Reply:
Let's add 1.19 in the build matrix, and only declare we support these two major versions.